geopolars/src/geoseries.rs

```diff
@@ -707,6 +759,27 @@ impl GeoSeries for Series {
         let series = Series::try_from(("result", Box::new(result) as ArrayRef))?;
         Ok(series)
     }
+
+    #[cfg(feature = "proj")]
+    fn to_crs(&self, from: &str, to: &str) -> Result<Series> {
```

Comment:
Can you move `to_crs` back to where it was for a smaller diff?

Reply:
sorry. Fixed